doc/check-options.md

```diff
@@ -0,0 +1,215 @@
+| ---------------- | :---------------------------------------------- | :------------------------------------------------------------------------------------ |
+| `parentSelector` | `button, [role=button], a[href], p, li, td, th` | Selector used to look for the parent of an image that could duplicate the image label |
+
+### label-content-name-mismatch
+
+| Option               | Default | Description                                                                                                                                                               |
+| -------------------- | :------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `pixelThreshold`     | `0.1`   | Percent of differences in pixel data or pixel width needed to determine if a font is a ligature font (ligature fonts are ignored when comparing the label to the content) |
+| `occuranceThreshold` | `3`     | Number of times the font is encountered before auto-assigning the font as a ligature or not                                                                               |
+
+### has-lang
+
+| Option       | Default                | Description                         |
+| ------------ | :--------------------- | :---------------------------------- |
+| `attributes` | `['lang', 'xml:lang']` | Attributes to check for lang values |
+
+### valid-lang
+
+| Option       | Default                | Description                               |
+| ------------ | :--------------------- | :---------------------------------------- |
+| `attributes` | `['lang', 'xml:lang']` | Attributes to check for valid lang values |
+
+### frame-tested
+
+| Option        | Default | Description                                                |
+| ------------- | :------ | :--------------------------------------------------------- |
+| `isViolation` | `false` | If an iframe without axe should be reported as a violation |
+
+### no-autoplay-audio
+
+| Option            | Default | Description                                                                   |
+| ----------------- | :------ | :---------------------------------------------------------------------------- |
+| `allowedDuration` | `3`     | Maximum time an audio is allowed to autoplay before being marked as violation |
+
+### css-orientation-lock
+
+| Option            | Default | Description                                                                                                                            |
+| ----------------- | :------ | :------------------------------------------------------------------------------------------------------------------------------------- |
+| `degreeThreshold` | `3`     | The difference of degrees from 180 and 90 that are considered orientation lock (for example, 93° is orientation locked but 94° is not) |
```

Comment:
Are we talking about a device orientation lock? The web API `ScreenOrientation.lock()`? The W3 Screen Orientation API? Please be specific.